ingeniamotion/motion.py

```diff
@@ -414,6 +416,36 @@ def get_actual_velocity(self, servo=DEFAULT_SERVO, axis=DEFAULT_AXIS):
         return self.mc.communication.get_register(self.ACTUAL_VELOCITY_REGISTER,
                                                   servo=servo, axis=axis)
 
+    def get_actual_direct_current(self, servo=DEFAULT_SERVO, axis=DEFAULT_AXIS):
+        """
+        Returns actual direct current register.
+
+        Args:
+            servo (str): servo alias to reference it. ``default`` by default.
+            axis (int): servo axis. ``1`` by default.
+
+        Returns:
+            float: actual direct current value
+
+        """
+        return self.mc.communication.get_register(self.ACTUAL_DIRECT_CURRENT_REGISTER,
+                                                  servo=servo, axis=axis)
+
+    def get_actual_quadrature_current(self, servo=DEFAULT_SERVO, axis=DEFAULT_AXIS):
```

Comment:
IMHO, `get_actual_direct_current `and `get_actual_quadrature_current` should be changed to `get_actual_current_direct` and `get_actual_current_quadrature` to be coerent with `set_current_direct` and `set_current_quadrature`